.rspec

```diff
@@ -1 +1,2 @@
---color
\ No newline at end of file
+--color
+--require spec_helper
```

Comment:
Maybe you should add this configuration: `--format documentation`, in order to make more readable when you run all tests.